modular_sand/code/modules/mining/machine_bluespaceminer.dm

```diff
@@ -1,3 +1,7 @@
+// Configuration defines
+#define BLUESPACE_MINER_BONUS_MULT		CONFIG_GET(number/bluespaceminer_mult_output)
+#define BLUESPACE_MINER_CRYSTAL_TIER	CONFIG_GET(number/bluespaceminer_mult_output)
```

Comment:
`number/bluespaceminer_crystal_tier`
...?

Reply:
This value was set mistakenly while propagating values, and will be fixed in a follow-up commit. Thank you for pointing out the issue.